docs/site/pdf_generation.md

```diff
@@ -1,50 +1,94 @@
 # PDF Generation of Problem Statements
-The DMOJ supports rendering problem statements to PDF. This can be useful in the case of on-site contests, where contestants
-are distributed paper versions of the problems.
+The DMOJ supports rendering problem statements to PDF. This can be useful in the case of on-site contests, where
+contestants are distributed paper versions of the problems.
 
-For example, [here](https://dmoj.ca/problem/ioi14p1/pdf) is a generated PDF of [this problem](https://dmoj.ca/problem/ioi14p1).
+For example, [here](https://dmoj.ca/problem/ioi14p1/pdf) is a generated PDF of
+[this problem](https://dmoj.ca/problem/ioi14p1).
 
-You can either use PhantomJS or SlimerJS to render PDFs.
+While Puppeteer is used for PDF generation in production, Selenium is the recommended PDF renderer.
```

Comment:
I think by definition something we don't use in production isn't as well-supported...

Reply:
I agree, but I'm not sure of a different way to mention it since there is no documentation. Recommending one renderer but documenting *only* another seems confusing.

Reply:
Reworded.